compiler-core/src/language_server/completer.rs

```diff
@@ -397,11 +398,18 @@ where
             });
         }
 
-        // Module types
+        // Module types and type aliases
```

Comment:
```suggestion
        // Module types
```

Reply:
I don't understand the proposed change. That section takes care of registering the module types and it's aliases.

Reply:
Type aliases are types.

Reply:
Uhm, makes sense. Will revert it!